Lumio updates (#228)
Lumi-supercomputer/lumi-userguide: docs/storage/lumio/connect-local.md

> # Using LUMI-O without connecting to LUMI

Reviewer: I would combine this page with "simple upload and download". It is basically the same, requires extra steps for configuration lumi-o modules can do on LUMI.

Author: I disagree with this one, for several reasons. The 'Access without connecting to LUMI' is now a bit short, but I was thinking to possibly add some guidance for Windows users also (at a later point after this PR). Also I don't think this is part of the most simple usage for people who are new to the whole concept of object storages, and possibly also with rclone. Also personally I like it that it is a separate use case example and well visible as a topic that "you can actually access LUMI-O without connecting to LUMI", since this is not self evident at all for a "basic user"

Author: Since no further comments, marking this as resolved

> # Using LUMI-O without connecting to LUMI

Reviewer: ```suggestion
# Using LUMI-O from your local computer
```

!!!Warning
    This example is for Linux and MacOS. For Windows machines this might not work similarily.

> This example is with `rclone`. The same works similarily with e.g `s3cmd` and other clients. (For s3cmd the default path under home directory is `~/.s3cfg`.)

Reviewer: ```suggestion
This example uses `rclone`. The same works similarly with e.g. `s3cmd` and other clients. (for the s3cmd the default path under home directory is `~/.s3cfg`.)
```

> By setting the file `rclone.conf` in the default path `~/.config/rclone/rclone.conf` (e.g. in your own laptop) with the correct configuration setting and valid access keys, it is possible to use rclone to access LUMI-O from anywhere across internet, e.g. from your own laptop. Connection to LUMI is not needed. 

Reviewer: ```suggestion
By setting the file `rclone.conf` (default config path is `~/.config/rclone/rclone.conf`) on your local system with the correct configuration setting and valid access key, rclone opens LUMI-O session from anywhere across internet, e.g. from your own laptop. Connection to LUMI is not needed. 
```

> Notice that you should have rclone installed for this example to work. If you don't have rclone installed, follow the directions e.g. [on rclone website](https://rclone.org/install/) to install rclone on your laptop.

Reviewer: ```suggestion
Note you should have rclone installed locally for this example to work. Follow the directions e.g. [on rclone website](https://rclone.org/install/) to install rclone on your local computer.
```

From auth.lumidata.eu you can get a valid rclone configuration for LUMI-O that is assosiated to (valid) access keys/tokens for a specific LUMI project. 

## Example: Create rclone config file and access LUMI-O directly from your laptop

**1)** Go to [auth.lumidata.eu](https://auth.lumidata.eu) 

**2)** Click the valid 'Access key' for your project (or if you don't have one, create a new key first for your project)

**3)** Select `rclone` from the Configuration templates and click 'Generate'

> **4)** Copy the created text block to a file (on your laptop) that you name 'rclone.conf' (or add it to an existing rclone.conf file). It's simplest to place it in the default path for rclone `~/.config/rclone/rclone.conf` 

Reviewer: ```suggestion
**4)** Copy the created text block to the local 'rclone.conf' file (create the file or add it to an existing file). Default path for rclone config is `~/.config/rclone/rclone.conf` 
```

**5)** Now you can access LUMI-O with rclone. Type e.g. 
```
rclone lsd lumi-46500XXXX-private:
```

> to see the buckets of your project 46500XXXX in LUMI-O. (_Replace 46500XXXX with your actual LUMI project number_.)

Reviewer: ```suggestion
to see existing buckets of your project 46500XXXX in LUMI-O. (_Replace `46500XXXX` with your actual LUMI project ID.)
```
